cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsInstanceConnector.java

```diff
@@ -131,25 +126,31 @@ public String getConsoleOutput(AuthenticatedContext authenticatedContext, CloudI
     )
     @Override
     public List<CloudVmInstanceStatus> reboot(AuthenticatedContext ac, List<CloudInstance> vms) {
-        AmazonEC2Client amazonEC2Client = awsClient.createAccess(new AwsCredentialView(ac.getCloudCredential()),
-                ac.getCloudContext().getLocation().getRegion().value());
-        List<CloudInstance> affectedVms = new ArrayList<>();
+        List<CloudVmInstanceStatus> rebootedVmsStatus = new ArrayList<>();
+
         try {
             if (!vms.isEmpty()) {
                 List<CloudVmInstanceStatus> statuses = check(ac, vms);
-                doReboot(affectedVms, amazonEC2Client, getStarted(statuses));
-                doStart(affectedVms, ac, getStopped(statuses));
-                logInvalidStatuses(getNotStoppedOrStarted(statuses));
+                doStop(new ArrayList<>(), ac, getStarted(statuses));
```

Comment:
`doStop` first parameter is never used, could you drop it?

Reply:
Thanks. I dropped it.